public/app/plugins/datasource/parca/QueryEditor/QueryEditor.test.tsx

```diff
@@ -13,7 +13,7 @@ describe('QueryEditor', () => {
   it('should render without error', async () => {
     setup();
 
-    expect(screen.findByText('process_cpu:cpu')).toBeDefined();
+    expect(await screen.findByText(/process_cpu - cpu/)).toBeDefined();
```

Comment:
Not certain how it was ever passing before as the screen has always been showing process_cpu - cpu and not process_cpu:cpu. My guess is that there is an issue with using async in the test callback where it often silently passes when the test case does not also use await due to timing of elements being rendered on the screen. I have seen this issue a few times around the repo but can't fully explain it :D

Reply:
Interesting, because I was wondering the same. Thanks for the info!

Reply:
Ah awesome work fixing this. I'd done the same [in this PR](https://github.com/grafana/grafana/pull/78261/files#r1450043963) where you can see my reasoning for why.